front/src/Components/Auth/index.js

```
@@ -0,0 +1,39 @@
+import React, { useContext } from 'react';
+import { GoogleLogin } from 'react-google-login';
+import sendRequest from '../../utils/SendRequest';
+import { LoggedInContext, ModalContext } from '../../Context';
+
+export default () => {
+  const { setLoggedIn } = useContext(LoggedInContext);
+  const { setOpen } = useContext(ModalContext);
+  const responseGoogle = async (response) => {
+    const tokenBlob = new Blob(
+      [JSON.stringify({ access_token: response.Zi.access_token }, null, 2)],
+      { type: 'application/json' },
+    );
+    const data = await sendRequest({
+      url: process.env.REACT_APP_AUTH_GOOGLE,
+      method: 'post',
+      body: tokenBlob,
+    });
+    setLoggedIn(data.result);
+    setOpen(!data.result);
+    localStorage.setItem('token', data.token);
+    localStorage.setItem('userImage', response.profileObj.imageUrl);
+    if (!data.result) {
```

`sendRequest`의 응답값 검증을 19행에서 진행하는 것은 어떨까요? 성공하지 않은 응답이 온 경우 실패한 케이스를 먼저 처리하는 것이 더 바람직한 방법일 때가 있습니다. "fail fast"라는 키워드로 검색해보시면 여러 자료들을 찾아볼 수 있습니다.